docs/core/diagnostics/dotnet-trace.md

```diff
@@ -0,0 +1,195 @@
+---
+title: dotnet-trace - .NET Core
+description: Installing and using the dotnet-trace command-line tool.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 10/01/2019
+---
+# Trace for performance analysis utility (`dotnet-trace`)
+
+**This article applies to:** .NET Core 3.0 SDK and later versions
+
+## Installing `dotnet-trace`
+
+To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+```dotnetcli
+dotnet tool install --global dotnet-trace
+```
+
+## Synopsis
+
+```console
+dotnet-trace [-h, --help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-trace` tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It's built around the cross-platform `EventPipe` technology of the .NET Core runtime. `dotnet-trace` delivers the same experience on Windows, Linux, or macOS.
+
+## Options
+
+- **`--version`**
+
+Display the version of the dotnet-counters utility.
+
+- **`-h|--help`**
+
+Show command-line help.
+
+## Commands
+
+| Command                                                     |
+| ----------------------------------------------------------- |
+| [dotnet-trace collect](#dotnet-trace-collect)               |
+| [dotnet-trace convert](#dotnet-trace-convert)               |
+| [dotnet-trace list-processes](#dotnet-trace-list-processes) |
+| [dotnet-trace list-profiles](#dotnet-trace-list-profiles)   |
+
+## dotnet-trace collect
+
+Collects a diagnostic trace from a running process.
+
+### Synopsis
+
+```console
+dotnet-trace collect [-h|--help] [-p|--process-id <pid>] [--buffersize <size>] [-o|--output <trace-file-path>]
+    [--providers <list-of-comma-separated-providers>] [--profile <profile-name>] [--format <NetTrace|Speedscope>]
+```
+
+### Options
+
+- **`-p|--process-id <pid>`**
+
+  The process to collect the trace from.
+
+- **`--buffersize <size>`**
+
+  Sets the size of the in-memory circular buffer in megabytes. Default 256 MB.
+
+- **`-o|--output <trace-file-path>`**
+
+  The output path for the collected trace data. If not specified it defaults to `trace.nettrace`.
+
+- **`--providers <list-of-comma-separated-providers>`**
+
+  A list of EventPipe providers to be enabled. These providers supplement any providers implied by the `--profile argument`. If there's any inconsistency for a particular provider, the configuration here takes precedence over the implicit configuration from the profile.
```

Comment:
to help with localization, I'd also add that is a comma separated list in the localized text:
```suggestion
  A comma-separated list of `EventPipe` providers to be enabled. These providers supplement any providers implied by `--profile argument`. If there's any inconsistency for a particular provider, the configuration here takes precedence over the implicit configuration from the profile.
```